chili-epfl/FROG
# Added sign up and login modal to the wiki

This PR adds the ability for the user to login or sign up from the wiki.

**Testing instructions**

- Create a wiki page and click on the gear on the right. A drop down with an option to create an account or login or logout will be displayed based on the user logged in. 
- Creating an account and logging in should have expected behaviour 
- The rest of the wiki functionality should be unchanged
